gc/base/MarkMap.hpp

```diff
@@ -102,6 +103,20 @@ class MM_MarkMap : public MM_HeapMap
 		return _heapMapBaseDelta + (slotIndex << _heapMapIndexShift);
 	}
 
+	/**
+	 * check MarkMap if there is any liveObjects in the Card
+	 * this function assumes that card covers exactly 512 bytes.
+	 * @param heapAddress has to be card size aligned
+	 */
+	MMINLINE bool
+	areAnyLiveObjectsInCard(void* heapAddress) const
+	{
+#if（8 != BITS_PER_BYTE || 9 != CARD_SIZE_SHIFT）
```

Comment:
I think Aleks meant:
```suggestion
#if（8 != BITS_PER_BYTE) || (9 != CARD_SIZE_SHIFT）
```

Reply:
the preprocessor seems not like curly braces in an #if condition

Reply:
Just checking,  did you mean to say "parentheses", which are '(' and ')', or do you really mean "curly braces", which are '{' and '}'?

Reply:
I mean "parentheses", which are '(' and ')'

Reply:
They should be ok, but what you used does not seem to be ordinary ASCII parentheses, even though they look same. They appear to be Unicode %uFF08, while ordinary ones are %28.

Copy/paste your line (or individual characters) here: https://www.online-toolz.com/tools/text-unicode-entities-convertor.php

Did you use non-US keyboard?

Reply:
ok, I will try again, I am using the laptop keyboard